CQFDKS/Quizlet/Utilities/IdGenerate.cs

```diff
@@ -0,0 +1,14 @@
+﻿using IdGen;
+
+namespace Quizlet.Utilities
+{
+    public static class IdGenerate
+    {
+        public static IdGenerator generator = new IdGenerator(0);
+
+        public static int GenerateID()
+        {
```

Comment:
A CreateId() a IdGen-ben long-ot ad vissza, így ez túcsordul mint állat, főleg hogy ez idő alapon generál ahogy néztem a forráskódját. Vagyis az ID amit ezzel generálsz nem a legjobb megoldás. Ennél akár egy Random.Shared.Next() is jobb lenne. De felmerül a kérdés: A szimpla növekményes ID miért nem jó?